server/src/infra/repositories/media.repository.ts

```diff
@@ -23,10 +25,25 @@ export class MediaRepository implements IMediaRepository {
   }
 
   async resize(input: string | Buffer, output: string, options: ResizeOptions): Promise<void> {
-    await sharp(input, { failOn: 'none' })
+    let colorProfile = options.colorspace;
+    if (options.colorspace !== Colorspace.SRGB) {
+      try {
+        const { space } = await sharp(input).metadata();
+        // if the image is already in srgb, keep it that way
+        if (space && space === Colorspace.SRGB) {
```

Comment:
There's no need for `space &&` here. Even if space is null, it still won't equal the constant.

Reply:
Done